apps/emqx_dashboard/src/emqx_dashboard_app.erl

```diff
@@ -31,7 +31,8 @@ start(_StartType, _StartArgs) ->
     Tables = lists:append([
         emqx_dashboard_admin:create_tables(),
         emqx_dashboard_token:create_tables(),
-        emqx_dashboard_monitor:create_tables()
+        emqx_dashboard_monitor:create_tables(),
+        emqx_dashboard_login_lock:create_tables()
```

Comment:
It's an overkill to create a table for this small feature.
Possible to make use of the `extra` field in the admin table? like the way TOTP is supported